xbmc/guilib/GUIFontCache.h

```diff
@@ -121,16 +122,12 @@ struct CGUIFontCacheHash
 template<class Position>
 struct CGUIFontCacheKeysMatch
 {
-  bool operator()(const CGUIFontCacheKey<Position> &a, const CGUIFontCacheKey<Position> &b) const
+  bool operator()(const CGUIFontCacheKey<Position>& a, const CGUIFontCacheKey<Position>& b) const
   {
-    return a.m_text == b.m_text &&
-           a.m_colors == b.m_colors &&
-           a.m_alignment == b.m_alignment &&
-           a.m_scrolling == b.m_scrolling &&
-           a.m_maxPixelWidth == b.m_maxPixelWidth &&
+    return a.m_text == b.m_text && a.m_colors == b.m_colors && a.m_alignment == b.m_alignment &&
+           a.m_scrolling == b.m_scrolling && a.m_maxPixelWidth == b.m_maxPixelWidth &&
            Match(a.m_pos, a.m_matrix, b.m_pos, b.m_matrix, a.m_scrolling) &&
-           a.m_scaleX == b.m_scaleX &&
-           a.m_scaleY == b.m_scaleY;
+           a.m_scaleX == b.m_scaleX && a.m_scaleY == b.m_scaleY;
```

Comment:
id argue this makes it harder to read.

Reply:
Added a clang ignore for here and below.